RFP2212-Toy-Story/atelier
# PR for Reviews Module

Latest changes:
1. Update how the image appears in image modal on RR module
2. Utilize the AvgStarsArray component to have consistent stars between review/average rating, and what will be shown on other modules
3. Some styling changes to align to Q&A module (e.g. buttons, hovering, etc.)
4. Turn off linter rule around no booleans for attributes (needed for shared Form components for RR and Q&A to be able to make some form inputs required vs. not).
